logger.js

```diff
@@ -65,6 +65,16 @@ Logger.prototype.error = function error(err) {
             fullType: err.fullType
         }));
 
+    } else if (err.type === 'tchannel.request.timeout') {
```

Comment:
Evidently not in all cases. It depends on whether TChannel produces the timeout locally, or whether the timeout comes from a peer over the wire.